Marlin/Configuration.h

```diff
@@ -87,7 +87,7 @@
 
 // Choose the name from boards.h that matches your setup
 #ifndef MOTHERBOARD
-  #define MOTHERBOARD BOARD_RAMPS_14_EFB
+  #define MOTHERBOARD BOARD_RUMBA
```

Comment:
Obviously we cannot push this PR with your configurations in it. The idea with PRs is to set them up exactly as they should be published for everybody.